Extension/package.nls.json

```diff
@@ -1,7 +1,8 @@
 {
     "c_cpp.command.configurationSelect.title": "Select a configuration...",
     "c_cpp.command.configurationProviderSelect.title": "Change configuration provider...",
-    "c_cpp.command.configurationEdit.title": "Edit configurations...",
+    "c_cpp.command.configurationEditJSON.title": "Edit configurations... (JSON)",
```

Comment:
@michelleangela I think the "..." are always supposed to be at the end of the command text, e.g. "Edit configurations (JSON)…"